README.md

```diff
@@ -235,6 +235,16 @@ an `include` key with a list of globs to specify specific files that should be c
 
 > Note: include defaults to `['**']`
 
+`nyc` relies on [`micromatch`](https://www.npmjs.com/package/micromatch) to deal with the
```

Comment:
I like the idea of adding this blurb, but let's keep it short and sweet:

    > `nyc` uses micromatch for glob expansions, you can read its documentation here.